core/modules/ip_checksum.cc

```diff
@@ -34,11 +34,20 @@
 #include "../utils/ether.h"
 #include "../utils/ip.h"
 
+const Commands IPChecksum::cmds = {
+    {"get_initial_arg", "EmptyArg", MODULE_CMD_FUNC(&IPChecksum::GetInitialArg),
+     Command::THREAD_SAFE},
+};
+
+CommandResponse IPChecksum::GetInitialArg(const bess::pb::EmptyArg &) {
+  return CommandSuccess();
```

Comment:
As Shinae mentioned elsewhere (offline) we can make this the default. There's a minor danger that someone will then forget to implement it when using a non-trivial initial arg, but I'm not sure if it's worth protecting against this case.

Reply:
Pushed a fixup with this. Isn't the cleanest, but I'm not sure we can do much better atm